src/ui/App/App.tsx

```diff
@@ -433,28 +448,28 @@ export interface AppProps {
 }
 
 export function App({ initialView, mode, inspect }: AppProps) {
+  const windowContext = WindowContext.get();
   const bodyClassList = useMemo(() => {
     const result = [];
-    if (templateData.windowContext === 'dialog') {
+    if (windowContext.isDialog()) {
```

Comment:
can we move windowContext inside useMemo?

Reply:
I think it is not relevant anymore, since `windowContext` is now a global constant ✅